docs/zh_CN/rest-server/API.md

```diff
@@ -716,12 +716,103 @@ Status: 404
     <br />*Response if a server error occured*
     
 
+Status: 500
+
+{ "code": "UnknownError", "message": "*Upstream error messages*" }
+
+    <br /><br />## API v2
```

Comment:
there's format issue, do not need `<br>` here

Reply:
It's due to error format in English, I will fix it.